### ISSUE TITLE
Exception in thread "main" java.lang.IllegalStateException

### ISSUE THREAD

**Issue by Enoughsdv** [terminalsin/skidfuscator-java-obfuscator#16]
Java Version:
![image](https://user-images.githubusercontent.com/67561225/147965044-a23a5d28-19a3-4d07-b6e2-c3eab566c2ce.png)

ScreenShots:
![image](https://user-images.githubusercontent.com/67561225/147965078-64288280-6b36-4005-943a-5f4c88cf11fa.png)
![image](https://user-images.githubusercontent.com/67561225/147965095-afe32a8e-20af-4078-a506-0b30440637b8.png)
![image](https://user-images.githubusercontent.com/67561225/147965116-756ef747-3137-46e2-bf2b-9e9f02c46a17.png)
![image](https://user-images.githubusercontent.com/67561225/147965144-551b0bae-15fd-4134-b979-8f1ff4e3728c.png)

**Describe the bug**
Basically i try to obfuscate a .jar i have to test the obfuscator but it throws the rror i sent, i use ubuntu 20.04

**Comment by terminalsin** [comment]
Fixed in f0c60ffb1d6bd1ee91c699d6dc326f962bec353e
